source/comply/compliance-export.rst

```diff
@@ -8,26 +8,29 @@ Compliance export
 
  <p class="mm-label-note">Also available in legacy Mattermost Enterprise Edition E20</p>
 
-Enterprise deployments with a requirement to archive history beyond the data retention period can export compliance reports to third-party systems. Integration with Actiance Vantage, Global Relay, and Proofpoint are currently supported.
+Mattermost Enterprise customers can archive history or transfer message data to third-party systems for auditing and compliance purposes with compliance exports. Supported integrations include `Actiance Vantage <#actiance-xml>`__, `Global Relay <#global-relay-eml>`__, and `Proofpoint <#proofpoint>`__. 
 
-By default, Mattermost stores all message history, providing an unlimited search history to admins and end users. In Mattermost Enterprise, you may set a :doc:`custom data retention policy </comply/data-retention-policy>` for how long messages and file uploads are kept in Mattermost channels and direct messages.
+.. tip::
 
-Compliance exports are produced from the System Console, containing all messages including:
+   While Mattermost stores all message history by default, system admins can specify a :doc:`custom data retention policy </comply/data-retention-policy>` for how long messages and file uploads are kept in Mattermost channels and direct messages.
+
+From Mattermost v10.5, compliance exports include performance improvements for large daily data sets with changes affecting output formats, system performance, and logic. Compliance exports provide compliance teams complete information to reconstruct the state of a channel, and to determine who had visibility on an initial message, or when the message was edited or deleted. Compliance teams can track a message by its MessageId as it is edited or deleted, and across batches and exports periods.
+
+Overview
+---------
+
+Compliance exports are produced from the System Console, and contain all messages including:
 
 - Messages sent in direct message channels
 - File uploads
 - Posts from plugins
 - Posts from bots/webhooks
 
-Exports include information on channel member history at the time the message was posted. 
+Exports include information on channel member history at the time the message was posted.
 
 - Entries for deleted messages and files are included in CSV and Actiance reports. The deleted content is included in the compliance export. 
 - Global Relay reports include file deletion entries but message deletion entries are excluded.
```

Comment:
We can remove this -- all export formats include deleted message info now.

Reply:
Removed.